### PR TITLE
Move test elfs to test dir

### PR DESCRIPTION
**Pull request by Megamouse**
TEST12345 is already taken by dev builds

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,6 +45,9 @@
 !/bin/soft_oal.dll
 !/bin/xaudio2_9redist.dll
 
+# Test Programs
+!/bin/test/
```
Comment (elad335) [inline .gitignore:49]: Hm why enforce ignoring its contents? shouldn't it allow .self/.elf file modifications?

**Comment by elad335** [comment]
I vote putting it outside of ganes' file system visibility as well, shouldn't be accessible by them and they can delete it in theory.

File: .gitignore (hunk 1 above)
Comment (Megamouse) [reply .gitignore:49]: It's the opposite 🤔

Comment (elad335) [reply .gitignore:49]: Carry on then.

**Comment by MSuih** [comment]
I don't think they are included in appimages anyways, are they?

**Comment by RainbowCookie32** [comment]
> I don't think they are included in appimages anyways, are they? People do seem to need to download these manually on Linux.
> 
> Or if they are then maybe they're not extracting properly.

If they are, I was never able to find them. Maybe it's hidden somewhere inside the AppImage itself, but dev_hdd0 never had them out of the box, I had to get them from GitHub or a Windows zip.

**Comment by kd-11** [comment]
> I don't think they are included in appimages anyways, are they? People do seem to need to download these manually on Linux.
> 
> Or if they are then maybe they're not extracting properly.

Then this is a bug.
While most developers don't care much about these, they are very useful when diagnosing user-specific problems. The first thing we ask users reporting issues is to run triangle test and cube test just to make sure they didn't mess up their graphics drivers too badly in the first place. This is a much easier way to get a layman to test a sample that allows us to eliminate game-specific weirdness.

In the end, you can put them wherever, it just needs to be easy to find. Or you can add some debug menu to the GUI to access them or something.

**Comment by Megamouse** [comment]
I don't feel like fixing a Leenux bug xD

**Comment by Megamouse** [comment]
Whatever floats your boat

**Comment by Nekotekina** [comment]
Maybe make a special boot menu for tests? That could work for appimages as well.

**Comment by Megamouse** [comment]
It's pointless to show tests in the game list.
It's useless for 99.9% of all users.
In my opinion it's enough to have them in some dir and boot them with the menu.
Of course we could add a menu option that opens the proper dir already.

**Comment by elad335** [comment]
Tests should run on all RPCS3 versions, making them online not only solves this problem but also allows an official way to use the latest hw tests to compare the performance of all RPCS3 builds in addition to saving RPCS3 package space.

**Comment by Megamouse** [comment]
That's completely missing the point of this PR

**Comment by Megamouse** [comment]
I don't care where they are i just want them out of my elevon dir xD

**Review by RipleyTom — APPROVED** [review]
A work of art! (I'll fix the linux thing in a separate PR)

**Comment by Vestrel** [comment]
Should close #11326

**Pull request merged**